config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -219,6 +219,7 @@ useHostNetwork: true
 
 # Daemonset DNS policy
 dnsPolicy: ""
+dnsConfig: []
```

Comment:
Can you also add this to the Helm chart README? https://github.com/aws/aws-node-termination-handler/blob/main/config/helm/aws-node-termination-handler/README.md?plain=1#L135